app/controllers/items_controller.rb

```diff
@@ -5,8 +5,23 @@ class ItemsController < ApplicationController
   #before_action :current_user.admin?         #WIP must verify that the user is an admin in order for to run any procudure in this controller. An admin
                                               #should be the only user that is able to add/edit/delete items in the database
   
-  def index                                   #shows all library items in database
-    @items = Item.all
+  def index                                   #items in library
+    @items = search
+    @page_title = "All library resources"
+    
+    if @items.nil?
+      # case when we don't use search, just list all
+      @items = Item.all
+      
+    elsif @items.empty?
```

Comment:
Since the search term has to be exact, it might be useful to also display a message suggesting that the user try something more specific.